forwarding.go

```diff
@@ -185,13 +195,12 @@ func (r *oauthProxy) forwardProxyHandler() func(*http.Request, *http.Response) {
 
 					// step: add some debugging
 					r.log.Info("successfully refreshed the access token",
-						zap.String("subject", state.identity.ID),
-						zap.String("email", state.identity.Email),
+						zap.String("subject", state.subject),
 						zap.String("expires", state.expiration.Format(time.RFC3339)))
 				} else {
 					r.log.Info("session does not support refresh token, acquiring new token",
-						zap.String("subject", state.identity.ID),
-						zap.String("email", state.identity.Email))
+						zap.String("subject", state.subject),
+						zap.String("email", state.subject))
```

Comment:
Did you mean to remove this part about the email, like for the other cases?

Reply:
state doesn't have identity after refactor and even isn't in standard claims thus and this info is just for logging, doesn't seem to have big importance, i will remove it from logging